<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Dr Amit Kahana</title>
    <link rel="stylesheet" href="style.css">
    <style>
        * {
            margin: 0;
            padding: 0;
            box-sizing: border-box;
        }
    </style>
    <script>
        function togglePublication(id) {
            var content = document.getElementById(id);
            content.style.display = content.style.display === 'none' ? 'flex' : 'none';
        }
    </script>
</head>
<body>
    <script src="https://use.typekit.net/bkt6ydm.js"></script>
    <script>try{Typekit.load({ async: true });}catch(e){}</script>

    <div class="main-container">
        <header class="title-card">
            <h1>Dr Amit Kahana</h1>
            <p class="subtitle">Origin of Life &nbsp;&nbsp;| &nbsp;&nbsp;
                Systems Chemistry &nbsp;&nbsp;| &nbsp;&nbsp;Molecular Biology
            </p>
                <p class="subtitle2">amit.kahana@glasgow.ac.uk
            </p>
        </header>

        <div class="sidebar">
            <!-- <h2>Dr Amit<br>Kahana</h2> -->
            <a href="#about">About Me</a>
            <a href="#research">Research</a>
            <a href="#publications">Publications</a>
            <a href="#associations">Associations</a>
            <a href="#contact">Contact</a>
            <!-- <div class="sidebar-icons">
                <a href="mailto:a.kahana.1@research.gla.ac.uk" target="_blank"><img src="./images/envelope-solid.svg"></a>
                <a href="https://scholar.google.co.il/citations?user=S1-Cuh0AAAAJ&hl=iw" target="_blank"><img src="./images/google-scholar-brands-solid.svg"></a>
                <a href="https://www.linkedin.com/in/amit-kahana-8b9281269/" target="_blank"><img src="./images/linkedin-in-brands-solid.svg"></a>
                <a href="https://x.com/AmitKahana" target="_blank"><img src="./images/twitter-brands-solid.svg"></a>
                <a href="https://www.researchgate.net/profile/Amit-Kahana?ev=hdr_xprf" target="_blank"><img src="./images/researchgate-brands-solid.svg"></a>
                <a href="https://bsky.app/profile/amitkahana.bsky.social" target="_blank"><img src="./images/bluesky-brands-solid.svg"></a>
            </div> -->
        </div>

        <div class="content">
            <section id="about" class="about">
                <h1 class="about-header">About Me</h1>
                <div class="about-content">
                    <div class="about-text">
                        <p>A protobiologist and biochemist interested in the emergence of life from complex chemical systems.
                            With a diverse experimental and computational background, I am passionate about innovation and 
                            discovery through ambitious multidisciplinary research.<br><br>
                            I obtained my BSc in Biology from Ben-Gurion University, and my MSc in 
                            Molecular Biology from the Weizmann Institute of Science. I conducted 
                            research into bacterial ecology, immunology, protein evolution and molecular genetics. 
                            Later, I studied the evolution of catalytic chemical networks in the lab of Prof. 
                            Doron Lancet, researching the origin of life in mixed lipid assemblies.<br><br>
                            I successfully completed my PhD in Chemistry from the University of Glasgow, under the supervision
                            of Prof. Lee Cronin. I investigated selection and evolution in chemical and 
                            biological systems, expanding on the Assembly Theory framework, and 
                            developed new analytical tools to uncover causal links at the molecular scale.<br><br>
                            My scientific goals include: mapping the informational and functional landscape of living 
                            systems; uncovering the fundamentals of selection and evolution; and generating 
                            self-reproducing protobiological systems in the lab.<br><br>
                            When I'm not looking into the origin of life, I enjoy music and podcasts, cooking vegan food, 
                            sports climbing, hiking, going on adventures and continually trying new things.<br><br>
                        </p>
                    </div>
                    <div class="about-image">
                        <img src="./images/amit2.jpg">
                        <!-- <p class="about-caption">a.kahana.1@research.gla.ac.uk</p> -->
                    </div>
                </div>
            </section>


            <section id="research">
                <h1>Research</h1>
                <div class="col-xs-10 col-xs-offset-1 col-sm-8 col-sm-offset-2">
                    <ul class="timeline timeline-split">
                        <li class="timeline-item period">
                            <div class="timeline-info"></div>
                            <div class="timeline-marker"></div>
                            <div class="timeline-content">
                                <h2 class="timeline-title" style="font-weight: normal;">
                                <strong>PhD in Chemistry | </strong><em>University of Glasgow</em></h2>
                            </div>
                        </li>
                        <li class="timeline-item">
                            <div class="timeline-info">
                                <span>2022 - 2025</span>
                            </div>
                            <div class="timeline-marker"></div>
                            <div class="timeline-content">
                                <h3 class="timeline-title">Selection in Complex Chemical Systems</h3>
                                <p>Researcher in the lab of Prof. Lee Cronin.<br>
                                    Investigated Assembly Theory as a fundamental framework for the study of selection and 
                                    evolution, through experimentation with advanced analytics and automated systems supported 
                                    by theoretical work.</p>
                            </div>
                        </li>
                        <li class="timeline-item period">
                            <div class="timeline-info"></div>
                            <div class="timeline-marker"></div>
                            <div class="timeline-content">
                                <h2 class="timeline-title" style="font-weight: normal;">
                                <strong>MSc in Molecular Biology | </strong><em>Weizmann Institute of Science</em></h2>
                            </div>
                        </li>
                        <li class="timeline-item">
                            <div class="timeline-info">
                                <span>2018-2022</span>
                            </div>
                            <div class="timeline-marker"></div>
                            <div class="timeline-content">
                                <h3 class="timeline-title">Micellar Origin of Life</h3>
                                <p>Researcher in the lab of Prof. Doron Lancet. <br>
                                    Researched the origin of life in a Lipid World scenario using advanced computational 
                                    modelling and kinetic simulations of self-reproducing complex catalytic networks.</p>
                            </div>
                        </li>
                        <li class="timeline-item">
                            <div class="timeline-info">
                                <span>2020-2020</span>
                            </div>
                            <div class="timeline-marker"></div>
                            <div class="timeline-content">
                                <h3 class="timeline-title">Functional Genomics</h3>
                                <p>Research Intern in the lab of Prof. Maya Schuldiner. <br>
                                    Predicted the functionality of unexplored enzymes using advanced structural homology, 
                                    and performed experimental validation.</p>
                            </div>
                        </li>
                        <li class="timeline-item">
                            <div class="timeline-info">
                                <span>2019-2020</span>
                            </div>
                            <div class="timeline-marker"></div>
                            <div class="timeline-content">
                                <h3 class="timeline-title">Protein Evolution</h3>
                                <p>Research Intern in the lab of Prof. Dan Tawfik. <br>
                                    Examined the functional properties and folding capacities of a primordial RNA-binding 
                                    protein candidate in the presence of polyamines.</p>
                            </div>
                        </li>
                        <li class="timeline-item period">
                            <div class="timeline-info"></div>
                            <div class="timeline-marker"></div>
                            <div class="timeline-content">
                                <h2 class="timeline-title" style="font-weight: normal;">
                                <strong>BSc in Biology | </strong><em>Ben-Gurion University</em></h2>
                            </div>
                        </li>
                        <li class="timeline-item">
                            <div class="timeline-info">
                                <span>2017-2019</span>
                            </div>
                            <div class="timeline-marker"></div>
                            <div class="timeline-content">
                                <h3 class="timeline-title">Avian Immunology</h3>
                                <p>Research Assistant in the lab of Prof. Tomer Hertz. <br>
                                    Profiled the antibody repertoires of avian species and identified exposures to 
                                    influenza and other viruses in relation to migration patterns of wild flocks.</p>
                            </div>
                        </li>
                        <li class="timeline-item">
                            <div class="timeline-info">
                                <span>2018-2019</span>
                            </div>
                            <div class="timeline-marker"></div>
                            <div class="timeline-content">
                                <h3 class="timeline-title">Microbial Ecology</h3>
                                <p>Research Intern in the lab of Prof. Itzik Mizrahi. <br>
                                    Explored the dynamics of Horizontal Gene Transfer in microbial communities of ruminants.</p>
                            </div>
                        </li>
                    </ul>
                </div>
            </section>

            <section id="publications">
                <h1>Publications</h1>
                <h2 style="color:#182633">Notable Projects:</h2>
                <div class="publication" onclick="togglePublication('pub2')">
                    <div class="pub-header">
                        <p class="status">2024</p>
                        <div class="pub-info">
                            <strong>Constructing the Molecular Tree of Life using Assembly Theory and Mass Spectrometry</strong>
                            <p><strong>A Kahana</strong>, A MacLeod, H Mehr, A Sharma, E Carrick, 
                                M Jirasek, S Walker and L Cronin</p>
                                <em>arXiv</em>
                        </div>
                    </div>                
                    <div id="pub2" class="publication-content" style="display: flex;">
                        <img src="./images/tol2.png" alt="Paper 2">
                        <p>We present a biochemistry-agnostic method to map evolutionary relationships at the molecular scale 
                            using mass spectrometry and Assembly Theory, without structure elucidation of analytes. 
                            By analyzing diverse biological and non-biological samples, we were able to infer joint assembly 
                            spaces of sampled species to determine their biogenicity and taxonomic grouping. 
                            We developed an assembly-based phylogenetic tree that aligns with the consensual 
                            genome-based model. Later we were able to use our approach to track bacterial lineages 
                            exhibiting phenotypic variation. Our results demonstrate we can expand causal molecular inference 
                            to non-sequence information without requiring exact molecular identities, thereby opening the 
                            possibility to study previously inaccessible biological domains.</p>
                    </div>
                </div>
                <div class="publication" onclick="togglePublication('pub3')">
                    <div class="pub-header">
                        <p class="status">2023</p>
                        <div class="pub-info">
                            <strong>Attractor Dynamics Drives Self-Reproduction in Protobiological Catalytic Networks</strong>
                            <p><strong>A Kahana</strong>, L Segev and D Lancet</p>
                            <em>CellRepPhysSci</em>
                        </div>
                    </div> 
                    <div id="pub3" class="publication-content" style="display: flex;">
                        <img src="./images/attractor.png" alt="Paper 3">
                        <p>The origin of life likely involved a shift from chaotic chemistry to self-reproducing structures. 
                            We examine the reproduction characteristics of the GARD lipid-based model, which simulates 
                            physicochemically rigorous, self-replicating networks. The model shows compatibility with 
                            heterogeneous environments and portrays trans-generational information transfer. However, we find 
                            that self-reproducing states are rare in compositional space, indicating that random exploration
                            would be inefficient to attain reproduction capacities. Rewardingly, we sow thatb all 
                            self-reproducing states are dynamic attractors of the catalytic network, suggesting a greatly 
                            enhanced propensity for reproduction and primal evolution, augmenting the likelihood of protolife 
                            appearance.</p>
                    </div>
                </div>
                <div class="publication" onclick="togglePublication('pub7')">
                    <div class="pub-header">
                        <p class="status">2021</p>
                        <div class="pub-info">
                            <strong>Self-Reproducing Catalytic Micelles as Nanoscopic Protocell Precursors</strong>
                            <p><strong>A Kahana</strong> and D Lancet</p>
                            <em>NatRevChem</em>
                        </div>
                    </div>
                    <div id="pub7" class="publication-content" style="display: flex;">
                        <img src="./images/natrevchem.png" alt="Paper 7">
                        <p>Protocells are often seen as bilayer-enclosed precursors of life, relying on replicating biopolymers 
                            for self-reproduction. This Perspective presents an alternative scenario where reproducing lipid 
                            micelles with catalytic abilities preceded biopolymer-containing protocells. Support for this idea 
                            comes from experiments on micellar catalysis and autocatalytic proliferation, as well as 
                            cross-catalysis in mixed micelles leading to life-like dynamics. Our chemically stringent 
                            computer-simulated model further shows how catalytic lipid networks could enable micellar 
                            compositional reproduction, facilitating primal selection and evolution. Finally, we highlight
                            studies on how endogenously catalysed lipid modifications could guide further protocellular 
                            complexification, supporting the idea that lipid micellar assemblies seeded protocellular evolution.</p>
                    </div>
                </div>
                <h2 style="color:#182633">Other Projects:</h2>
                <div class="publication" onclick="togglePublication('pub1')">
                    <div class="pub-header">
                        <p class="status">In Preparation</p>
                        <div class="pub-info">
                            <strong>Selectivity of Catalytic Interactions Delineates Directed Exploration in Molecular Assembly Spaces</strong>
                            <p><strong>A Kahana</strong>, M Jirasek and L Cronin</p>
                            <em></em>
                        </div>
                    </div>
                    <div id="pub1" class="publication-content" style="display: none;">
                        <img src="./images/selectivity.png" alt="Paper 1">
                        <p>A living system is a selective chemical network that evolves from simple origins. 
                            To promote evolution, protocellular networks must exhibit sufficient selectivity in their 
                            molecular interactions, guiding molecular assembly. To test this, we conducted over 5600 
                            organocatalytic reactions in two chemical systems, revealing that greater molecular assembly 
                            reduce average catalytic selectivity, highlighting a trade-off between exploration and selection. 
                            Both systems also showed a selectivity barrier as complexity increased, explaining the gap between living and non-living systems.
                            This work provides insight into the kinetic foundations of selection in catalytic networks and the 
                            transition from chemistry to life.</p>
                        </div>
                </div>
                <div class="publication" onclick="togglePublication('pub4')">
                    <div class="pub-header">
                        <p class="status">2022</p>
                        <div class="pub-info">
                            <strong>Micellar Composition Affects Lipid Accretion Kinetics in Molecular Dynamics Simulations: Support for Lipid Network Reproduction</strong>
                            <p><strong>A Kahana</strong>, D Lancet and Z Palmai</p>
                            <em>Life</em>
                        </div>
                    </div> 
                    <div id="pub4" class="publication-content" style="display: none;">
                        <img src="./images/md.png" alt="Paper 4">
                        <p>Mixed lipid micelles, with their growth dynamics and catalytic properties, were proposed to 
                            facilitate the origin of life. Our previous research suggested micellar self-reproduction 
                            involves lipid accretion, leading to compositional homeostasis. Using Molecular Dynamics 
                            simulations, we tracked micellar accretion from 54 lipid monomers to study self-assembly in 
                            mixed lipid clusters. We observed significant rate changes based on composition and size, 
                            revealing the underlying kinetic mechanisms and energy contributions. Our findings support the 
                            potential for compositional homeostasis in mixed micelles, reinforcing the basis for micellar 
                            self-reproduction and its relevance to the origin of life.</p>
                    </div>
                </div>
                <div class="publication" onclick="togglePublication('pub5')">
                    <div class="pub-header">
                        <p class="status">2022</p>
                        <div class="pub-info">
                            <strong>A Similarity-Based Method for Predicting Enzymatic Functions in Yeast Uncovers a New AMP Hydrolase</strong>
                            <p><strong>A Kahana</strong>*, N Cohen* and M Schuldiner</p>
                            <em>JMB</em>
                        </div>
                    </div>
                    <div id="pub5" class="publication-content" style="display: none;">
                        <img src="./images/yeast.png" alt="Paper 5">
                        <p>Despite the complete genomic sequencing of Saccharomyces cerevisiae, a significant portion remains 
                            functionally unannotated, limiting our understanding of cellular activity. Recent advancements in 
                            genomic and structural data, along with new computational methods, now allow us to extract 
                            meaningful insights into the functional identity of uncharacterized proteins in yeast. In this work,
                            We created a database of sequence and structural similarity predictions to identify candidate 
                            enzymes for unknown biochemical reactions. Using this approach, we discovered that Yhr202w, now 
                            named Smn1 (Scavenger MonoNucleotidase 1), functions as an extracellular AMP hydrolase in the NAD 
                            degradation pathway. Our new methodology provides a framework, that can be adopted to other organisms, 
                            for uncovering new enzymatic functions of uncharacterized proteins.</p>
                    </div>
                </div>
                <div class="publication" onclick="togglePublication('pub6')">
                    <div class="pub-header">
                        <p class="status">2022</p>
                        <div class="pub-info">
                            <strong>FLU-LISA: High Throughput Antibody Profiling Using Antigen Microarrays</strong>
                            <p>S Levy, M Abd Alhadi, A Azulay, <strong>A Kahana</strong>, N Bujanover, R Gazit, M A McGargill,
                                L M Friedman and T Hertz</p>
                            <em>ICB</em>
                        </div>
                    </div>
                    <div id="pub6" class="publication-content" style="display: none;">
                        <img src="./images/flu.png" alt="Paper 6">
                        <p>Vaccination and natural infection trigger humoral immune responses, with antibodies encoding 
                            immune histories of individuals. While most immunoassays measure responses to a single antigen, 
                            we introduce FLU-LISA, a high-throughput antigen microarray-based assay for rapid, simultaneous 
                            antibody profiling of IgG, IgA, and IgM against multiple antigens. Using influenza and SARS-CoV-2 
                            antigen microarrays, we demonstrated the sensitivity and specificity of FLU-LISA, compared to the 
                            traditional ELISA, in mice, chickens, and humans. FLU-LISA can also use dried blood spots, 
                            allowing profiling of hundreds of samples in one day, making it a promising alternative to ELISA.</p>
                    </div>
                </div>
                <div class="publication" onclick="togglePublication('pub8')">
                    <div class="pub-header">
                        <p class="status">2021</p>
                        <div class="pub-info">
                            <strong>Dynamic Lipid Aptamers: Non-Polymeric Chemical Path to Early Life</strong>
                            <p><strong>A Kahana</strong>, S Maslow and D Lancet</p>
                            <em>ChemSocRev</em>
                        </div>
                    </div>
                    <div id="pub8" class="publication-content" style="display: none;">
                        <img src="./images/aptamer.png" alt="Paper 8">
                        <p>The traditional view that the origin of life requires biopolymers like RNA and proteins is 
                            challenged by the improbability of their spontaneous formation in primordial chemistry. 
                            An alternative “Lipid-First” hypothesis suggests that lipid assemblies can spontaneously emerge, 
                            grow, and self-reproduce. A key question remains whether these lipid assemblies can engage in 
                            protein-like stereospecific interactions, essential for life processes. This Viewpoint highlights 
                            experimental evidence showing that lipid aggregates can have dynamic surface configurations 
                            capable of stereospecific molecular recognition, supporting the idea that lipids may have played 
                            a crucial role in the origin of life.</p>
                    </div>
                </div>
                <div class="publication" onclick="togglePublication('pub9')">
                    <div class="pub-header">
                        <p class="status">2020</p>
                        <div class="pub-info">
                            <strong>Polyamines Mediate Folding of Primordial Hyperacidic Helical Proteins</strong>
                            <p>D Despotović, L M Longo, E Aharon, <strong>A Kahana</strong>, T Scherf, I Gruic-Sovulj
                                and D S Tawfik</p>
                            <em>Biochemistry</em>
                        </div>
                    </div>                
                    <div id="pub9" class="publication-content" style="display: none;">
                        <img src="./images/polyamine.png" alt="Paper 9">
                        <p>Polyamines are known to stabilize nucleic acids by promoting folding, but their role in protein 
                            structure and function is less clear. Early proteins, thought to be highly acidic due to a lack 
                            of basic amino acids, might have relied on polyamines as chemical chaperones. In our study, we 
                            replaced all lysines of an ancestral 60-residue helix-bundle protein with glutamates, making 
                            it disordered. Polyamines induced folding at submillimolar concentrations, with their 
                            effectiveness increasing with the number of amine groups. In comparison with simple cations, 
                            polyamines were far more potent than Na+, while Mg2+ and Ca2+ were similar to diamines. 
                            We suggest that polyamines may have helped fold early acidic proteins and could influence 
                            protein regulation today through coil–helix transitions.</p>
                    </div>
                </div>
                <div class="publication" onclick="togglePublication('pub10')">
                    <div class="pub-header">
                        <p class="status">2019</p>
                        <div class="pub-info">
                            <strong>Enceladus: First Observed Primordial Soup Could Arbitrate Origin-of-Life Debate</strong>
                            <p><strong>A Kahana</strong>, P Schmitt-Kopplin and D Lancet</p>
                            <em>Astrobiology</em>
                        </div>
                    </div> 
                    <div id="pub10" class="publication-content" style="display: none;">
                        <img src="./images/enceladus.png" alt="Paper 10">
                        <p>A recent study has identified complex organic molecules in the plumes from Enceladus' subglacial 
                            ocean, suggesting a source of carbon-rich monomers and polymers. These findings point to the 
                            possibility of a prebiotic soup in Enceladus, providing an environment fitting for life's origin. 
                            The observed monomers resemble simple lipids, while the polymers are similar to terrestrial 
                            kerogens and organic compounds in carbonaceous meteorites. The authors propose that these polymers 
                            could break down into micelle-forming amphiphiles. This supports the Lipid World hypothesis, 
                            where life originated from mixed lipid assemblies, unlike other models requiring highly-specific 
                            chemistries.</p>
                    </div>
                </div>
                <div class="scholar">
                    <p>For a full list, see my 
                        <a href="https://scholar.google.co.il/citations?user=S1-Cuh0AAAAJ&hl=iw" target="_blank">
                            Google Scholar</a> profile.</p>
                </div>
            </section>

            <section id="associations">
                <h1>Associations</h1>
                <div class="associations">
                    <a href="https://www.chem.gla.ac.uk/cronin/" target="_blank"><img src="./images/cronin.png" alt="Cronin Lab"></a>
                    <a href="https://www.oolen.org/" target="_blank"><img src="./images/oolen.png" alt="OoLEN"></a>
                </div>
            </section>

            <section id="contact">
                <h1>Contact</h1>
                <div class="contact-icons">
                    <a href="mailto:a.kahana.1@research.gla.ac.uk" target="_blank" alt="Send Email"><img src="./images/envelope-solid.svg"></a>
                    <a href="https://scholar.google.co.il/citations?user=S1-Cuh0AAAAJ&hl=iw" target="_blank"><img src="./images/google-scholar-brands-solid.svg"></a>
                    <a href="https://www.linkedin.com/in/amit-kahana-8b9281269/" target="_blank"><img src="./images/linkedin-in-brands-solid.svg"></a>
                    <a href="https://x.com/AmitKahana" target="_blank"><img src="./images/twitter-brands-solid.svg"></a>
                    <a href="https://www.researchgate.net/profile/Amit-Kahana?ev=hdr_xprf" target="_blank"><img src="./images/researchgate-brands-solid.svg"></a>
                    <a href="https://bsky.app/profile/amitkahana.bsky.social" target="_blank"><img src="./images/bluesky-brands-solid.svg"></a>
                </div>
                <!-- <div class="contact-boxes"> -->
                    <!-- <a href="mailto:a.kahana.1@research.gla.ac.uk">Email</a> -->
                    <!-- <a href="https://scholar.google.co.il/citations?user=S1-Cuh0AAAAJ&hl=iw" target="_blank">Google Scholar</a> -->
                    <!-- <a href="https://www.linkedin.com/in/amit-kahana-8b9281269/" target="_blank">LinkedIn</a> -->
                    <!-- <a href="https://x.com/AmitKahana" target="_blank">Twitter</a> -->
                    <!-- <a href="https://www.researchgate.net/profile/Amit-Kahana?ev=hdr_xprf" target="_blank">ResearchGate</a> -->
                    <!-- <a href="https://bsky.app/profile/amitkahana.bsky.social" target="_blank">Bluesky</a> -->
                <!-- </div> -->
            </section>
        </div>
    </div>
</body>
</html>
